core/templates/search/search_pages_results.html

```diff
@@ -254,7 +254,7 @@ <h3 class="results">
     {% endif %}
 
       <div class="col-md-3">
-        <a href="{{ object.highlight_url }}" class="thumbnail">
+        <a href="{{ object.url }}" class="thumbnail">
```

Comment:
I think we should update the `highlight_url` behavior if we don't want the pagination information and such, but we need to keep the words that should be highlighted or else they won't show up on the individual page view

Reply:
Oh, I totally missed that. Was not intentional. Good catch.

I _think_ I'll probably just want to add a separate function for this purpose, as I think it's used to preserve pagination etc in other places. Maybe I'll rename this one `pagination_url` or similar and create another that does only preserve the `words` parameter. I'll ask for input if it doesn't seem like a straightforward change. :)